packages/reactivity/src/computed.ts

```diff
@@ -30,7 +30,7 @@ export class ComputedRefImpl<T> {
   public readonly effect: ReactiveEffect<T>
 
   public readonly __v_isRef = true
-  public readonly [ReactiveFlags.IS_READONLY]: boolean
+  public readonly [ReactiveFlags.IS_READONLY]: boolean = false
```

Comment:
Seems typescript 4.7.x is not able to recognise this variable being set on the `constructor`, just in case I've initialised here